Formula/elvish.rb

```diff
@@ -16,10 +16,12 @@ class Elvish < Formula
 
   def install
     ENV["GOPATH"] = buildpath
+    version = `git describe --tags --always`.strip
```

Comment:
> Update: as discussed

Can you elaborate on this?

> so that a full checkout is done and the version is generated correctly.

If this change is intended to be scoped to `HEAD` versions only it needs a bit of a rewrite to achieve that. If it is intended to be applied to both stable & `HEAD` specs then it won't work as expected because GitHub strips `.git` folders out of archive tarballs automatically IIRC.

Reply:
Using `#{version}` is sufficient. We should not switch to the tag just for the sake of retrieving git info here.